src/modules/account/RadixDataAccountSystem.ts

```diff
@@ -5,6 +5,7 @@ import { filter } from 'rxjs/operators'
 import RadixAccountSystem from './RadixAccountSystem'
 import RadixApplicationDataUpdate from './RadixApplicationDataUpdate'
 import RadixApplicationData from './RadixApplicationData'
+import RadixKeyPair from '../wallet/RadixKeyPair'
```

Comment:
Good catch, yes. In fact, visual studio code's auto import should do the correct thing automatically, no @murraco ?

Reply:
You're right here, we could import everything as you mentioned and it should be equivalent, in my case I personally prefer to import it line by line but just because of personal preferences. Maybe we should define a way of doing it and be consistent, what do you think @MuncleUscles?

And regarding circular dependencies, I think Edgars is working on that with the new atom model implementation.

Reply:
It is not equivalent, everything atom model related(everything in RadixAtomModel) needs to be imported from RadixAtomModel. In this particular case it happens to be fine either way, but there are cases where it's not, the RadixAtomModel file ensures the correct order of imports

It will be the same in the new atom model, except all the atom model stuff will be nicely organised into a single atommodel folder, so there's less confusion